packages/interface/src/components/JoinButton.tsx

```diff
@@ -233,6 +234,20 @@ export const JoinButton = (): JSX.Element => {
         try {
           const parsedPCD = (JSON.parse(result.pcdStr) as jsonPCD).pcd;
           const pcd = await ZKEdDSAEventTicketPCDPackage.deserialize(parsedPCD);
+
+          const response = await fetch("/api/poap", {
```

Comment:
thinking about should we made it consistent with other api calls, but if this works, since this is just a temporate PR that won't be merged to main, then I'm ok with it.

Reply:
yes agree, this is only for this round.